.travis.yml

```diff
@@ -8,6 +8,7 @@ rvm:
   - 1.9.3
   - 1.9.2
   - 2.0.0
+  - 2.1.6
```

Comment:
I don't think anyone uses this, so I don't see an issue with removing. I'm pushing for removing this gem altogether from our apps if possible.